hcl2template/testdata/complete/build.pkr.hcl

```diff
@@ -116,14 +116,15 @@ build {
         }
     }
 
-    post-processor "amazon-import" { 
+    post-processor "amazon-import" {
```

Comment:
What other post-processors is this needed for?

Reply:
I'm not sure if I understand the question here. So I'll give some infos:

The contents of this post-processor is sort of a mock struct containing all different types I'd like HCL2 to support:

https://github.com/hashicorp/packer/blob/a8fcb2d91a0d69b4b8a2f35e86d720d9538ef77d/hcl2template/internal/mock.go#L14-L39

And here `keep_input_artifact` is a common field so if it works here it works everywhere.

Reply:
Oops, this was a question to myself that I forgot to remove. When I review I usually leave questions for myself that I try to answer as I work through the review. This question was more about me understanding if this should be tested with other post processors. But I see that it doesn’t matter as the functionality is the same for all. Sorry for the confusion. But thanks for the additional context it confirms my understanding.